src/assets/guide-markdown/data-requirements.md

```diff
@@ -62,10 +48,11 @@ There are three main sources of data considered eligible for the Human Cell Atla
 To contribute data to the Human Cell Atlas Data Coordination Platform, follow this checklist:
 
 1. Make sure your dataset conforms to the suitability criteria above.
-2. Get informed about the data access type for your dataset (Open access, Controlled access, or a mix of both).  E.g. Data from living donors is always Controlled access.
-3. Prepare the data files and metadata for your dataset, based on the requirements for your data access type.
-4. Fill in the [HCA dataset questionnaire](https://docs.google.com/forms/d/e/1FAIpQLSdjPk2Z6xYozds53ycvXo57PvFsyqOF6XMpSWCVNTpQYalZzQ/viewform) to inform us of dataset details.
-5. Contact HCA Data Coordination Platform [data wrangling team](mailto:wrangler-team@data.humancellatlas.org?subject=get%20started%20with%20data%20contribution) to get you started.
+2. Get informed about the data access type for your dataset (Open access, Controlled access, or a mix of both).
+3. Sign the DCA agreement if required
```

Comment:
can we add either a link to the DCA to preview it or an email to the wranglers to ask about the DCA?

Reply:
That's valid. We should re-direct to an online DCA file that will be updated.

Reply:
Since DCA is expected to be changed in the near future, we won't add a preview to the DCA yet. Add email and prompt to ask wranglers about it.